### PR TITLE
Add support for AT Internet

### PR DESCRIPTION
**Pull request by BenDz**

### CODE REVIEW

**Comment by cramforce** [comment]
Looks good, please squash commits.
https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by rudygalfi** [comment]
Did you see my note about adding `${ampdocUrl}` and `${canonicalUrl}` to your requests? Without this change, the only information you'll have to know the source page of the requests would be a referrer, which could not work in some cases.

For example, consider the URL https://cdn.ampproject.org/c/washingtonpost.com/amphtml/news/wonk/wp/2016/02/08/the-fairest-and-funniest-way-to-split-the-dinner-bill/, which is a Washington Post article served by the AMP Cache. With just a referrer-based tracking, you would attribute the traffic to cdn.ampproject.org. However, in this case, the appropriate attribution is to washingtonpost.com, which you can get via either of these variables: `${ampdocUrl}` and `${canonicalUrl}`.

**Comment by BenDz** [comment]
Hi @rudygalfi - Yes, but unfortunately, we can't process it for the moment. I need to talk about that with our teams.
Thanks for the feedback anyway!

**Comment by cramforce** [comment]
@BenDz There is one lint error (See travis). Please fix and this is good to go.

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -56,6 +56,40 @@ export const ANALYTICS_CONFIG = {
...
+      'click': '${base}&' +
+        'pclick=${title}&' +
+        's2click=${level2}&' +
+        'p=${label}&' +
+        's2=${level2Click}&' +
+        'type=click&click=${type}${suffix}'
+    }
+  },
+
+  'atinternet': {
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: there seem to be two copies of the config. Can you remove one of them?

**Comment by amphtml-team** [comment]
@BenDz Wondering if you've had a chance to consider the proposed revision
yet? Any update you can offer? Thanks!

**Comment by BenDz** [comment]
@amphtml-team We just had a meeting, and we won't push `${ampdocUrl}` or `${canonicalUrl}` for the moment. The current version can be released. Thanks!

**Comment by amphtml-team** [comment]
@BenDz that sounds ok. Did you check the lint error that @cramforce
mentioned?

**Comment by BenDz** [comment]
@amphtml-team This should be good in my last commit https://github.com/BenDz/amphtml/commit/4d5a5d02f90db9326e05618be30968316d7cf637

**Comment by rudygalfi** [comment]
@BenDz Thanks, see that. Can you rebase? It looks like there are conflicts to resolve.

**Comment by rudygalfi** [comment]
@BenDz And also squash your commits? https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by rudygalfi** [comment]
@BenDz We merged your commit here: https://github.com/ampproject/amphtml/commit/4bfd2b55d70b1aae3fb715bf477d41d8cdfa9c15, so closing this pull request.

Please disregard my previous comments. We're all set here. Thanks!

**Comment by BenDz** [comment]
@rudygalfi Thanks!

**Pull request closed without merging**